src/scribe_data/check/check_language_metadata.py

```diff
@@ -0,0 +1,231 @@
+import difflib
+import json
+from pathlib import Path
+import sys
+
+LANGUAGE_DATA_EXTRACTION_DIR = Path(__file__).parent.parent / "language_data_extraction"
+
+LANGUAGE_METADATA_FILE = (
+    Path(__file__).parent.parent / "resources" / "language_metadata.json"
+)
+
+DATA_TYPE_METADATA_FILE = (
+    Path(__file__).parent.parent / "resources" / "data_type_metadata.json"
+)
+
+try:
+    with LANGUAGE_METADATA_FILE.open("r", encoding="utf-8") as file:
+        language_metadata = json.load(file)
+        languages_in_metadata = {
+            lang["language"]: {"iso": lang["iso"], "qid": lang["qid"]}
+            for lang in language_metadata["languages"]
+        }  # current language metadata
+
+        # languages_in_metadata = { # proposed language metadata
+        #     key.lower(): value for key, value in language_metadata.items()
+        # }  # Normalize keys to lowercase for case-insensitive comparison
+
+except (IOError, json.JSONDecodeError) as e:
+    print(f"Error reading language metadata: {e}")
+
+try:
+    with DATA_TYPE_METADATA_FILE.open("r", encoding="utf-8") as file:
+        data_type_metadata = json.load(file)
+        all_data_types = tuple(data_type_metadata.keys())
+
+except (IOError, json.JSONDecodeError) as e:
```

Comment:
Also, the `cli_utils.py` has loaded the `language_metadata.json` and `data_type_metadata.json` files for us. You could ignore loading it in here

Reply:
<img width="1027" alt="Screenshot 2024-10-16 at 15 13 42" src="https://github.com/user-attachments/assets/12c3f8cb-01ad-471d-961d-c40f101a5bac">

see the `cli_utils.py` contents

Reply:
does it mean we can directly use it? how?